packages/@react-native-windows/tester/package.json

```diff
@@ -29,7 +29,7 @@
     "@types/node": "^14.14.22",
     "eslint": "^7.32.0",
     "just-scripts": "^1.3.3",
-    "react-native": "0.70.0-rc.4",
+    "react-native": "0.70.0",
```

Comment:
Should we remove it? Or at least it must be the same version as above: "^0.70.0".

Reply:
According to Nick's notes on the release process and what we've done historically, this dep stays just 0.70.0